build.gradle

```diff
@@ -235,3 +238,20 @@ bootJar {
     attributes('Implementation-Version': project.version.toString())
   }
 }
+
+import org.flywaydb.gradle.task.FlywayMigrateTask
+
+flyway {
+  user = System.getenv('DB_USER')
+  password = System.getenv('DB_PASSWORD')
+
+  baselineOnMigrate = true
+  baselineVersion = '000'
+}
```

Comment:
i don't know whether we need this at all. can you please verify? the library executes this command for db migration:

```
gradle("-Pdburl='${dbHost}:${dbPort}/${dbName}?ssl=true&sslmode=require' -Pflyway.user='${dbUser}' -Pflyway.password='${dbPass}' migratePostgresDatabase")
```

so url, user and password are passed through to this task

Reply:
You mean, we can completely remove this?

Reply:
my guess would be yes. but can't be for sure :grimacing:

Reply:
so how did the test go on this one - can we completely remove `flyway` section?